en/reference/annotations-mapping.rst

```diff
@@ -49,6 +49,10 @@ Document
 Optional attributes:
 
 -  **nodeType**: PHPCR type for this node, default ``nt:unstructured``.
+-  **uniqueNodeType**: If this document uses a unique node type, set to ``true``
```

Comment:
Okay, I will add a section to the "Tools" page on the verification command and link back to that from here as well.